.github/workflows/release_manifest.yml

```diff
@@ -2,8 +2,15 @@ name: Release Installation Manifest
 
 on:
   workflow_dispatch:
+    inputs:
+      custom_tag_prefix:
+        description: "Custom tag"
+        required: false
+        default: ""
```

Comment:
```suggestion
        default: "manifest"
```

Reply:
The inputs option is under `workflow_dispatch` and this Is a special case. so I call it `custom_tag_prefix`